Sources/SwiftProtobuf/BinaryEncodingVisitor.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 /// Visitor that encodes a message graph in the protobuf binary wire format.
 internal struct BinaryEncodingVisitor: Visitor {
 
-  private var encoder: BinaryEncoder
+  var encoder: BinaryEncoder
```

Comment:
I was thinking since it is used in the other type at the end, it did, but maybe not.  But I'm about to use it in the next PR for some validation from another file.